src/stratis_cli/_errors.py

```diff
@@ -427,3 +427,28 @@ def __str__(self):
             self.minimum_version,
             self.maximum_version,
         )
+
+
+class StratisCliAggregateError(StratisCliRuntimeError):
+    """
+    Raised when multiple errors have occured in a looping operation.
+    """
+
+    def __init__(self, operation, type_, errors):
+        """
+        Initializer.
+        :param str operation: the looping operation that failed.
+        :param str type_: the type of the resource for which the operation failed.
+        :param errors: list of all errors that occurred during the looping operation.
+        :type errors: list of Exception
+        """
+        # pylint: disable=super-init-not-called
+        self.operation = operation
+        self.type = type_
+        self.errors = errors
+
+    def __str__(self):
+        return (
+            "The operation '%s' on a resource of type %s failed. The following"
+            "errors occurred:\n%s"
+        ) % (self.operation, self.type, "\n".join(self.errors))
```

Comment:
Please use os.linesep instead of "\n" here.

Also, you'll want to stringify each error in ```self.errors```, otherwise you'll get a ```TypeError``` on execution.